src/schedulers/unstaked/unstaked.service.ts

```diff
@@ -100,7 +100,7 @@ export class UnstakedService implements OnModuleInit {
         }
       }
     } catch (err) {
-      this.logger.error(`unstaked error ${err}`);
+      this.logger.log(`unstaked error ${err}`);
```

Comment:
Instead of turning the `error` log into a regular `log` can you mock the `Logger` class instead so it won't log anything? 

Maybe by doing something like this:
```
jest.mock('@nestjs/common', () => ({
  Logger: jest.fn(name => { //eslint-disable-line
    return {
      log: jest.fn(),
    }
  }),
}));
```

Also, please spy on the `Logger` using `jest.spyOn` to check if the number of logging is correct.

Reply:
Oke.

But using jest.mock is not work, so i tried other way to handle logger.